licenses.d2iq.yaml

```diff
@@ -579,10 +579,10 @@ resources:
       - url: https://github.com/kubernetes/kubectl
         ref: v0${image_tag#1}
         license_path: LICENSE
-  - container_image: registry.k8s.io/ingress-nginx/kube-webhook-certgen:v20221220-controller-v1.5.1-58-g787ea74b6
+  - container_image: registry.k8s.io/ingress-nginx/kube-webhook-certgen:v1.4.3
     sources:
       - url: https://github.com/kubernetes/ingress-nginx
-        ref:  controller-v1.5.2
+        ref: ${image_tag}
```

Comment:
I've checked the repo and the image is versioned in a static file in the repo. We can use the `controller-v1.11.2` ref instead.

Reply:
done